package.json

```diff
@@ -22,17 +22,18 @@
   "files": [
     "dist",
     "api-specifications",
-    "core"
+    "core",
+    "sn-action-button-ext"
   ],
   "engines": {
     "node": ">=16"
   },
   "main": "dist/sn-action-button.js",
   "module": "dist/sn-action-button.esm.js",
   "scripts": {
-    "build": "node ./tools/build.js --core && shx cp assets/* dist",
-    "build:dev": "node ./tools/build.js --core --mode development && shx cp assets/* dist",
-    "build:watch": "node ./tools/build.js --mode development -w",
+    "build": "node ./tools/build.js --core --ext && shx cp assets/* dist",
```

Comment:
the files in assets can also be generated by nebula sense, right? So do we still need assets?

Reply:
there is an example picture in there, i think it still should be there

Reply:
right, preview image can not be generated by sense.

Reply:
I had a try in this branch.

<img width="261" alt="Screenshot 2021-12-20 at 14 54 44" src="https://user-images.githubusercontent.com/4471489/146778526-818cdace-733c-4903-9c56-e0b3b249e6d7.png">
the preview.gif is not copied into sn-action-button-ext. Not sure why. I do see preview.gif in assets and dist folders.